setup/config/webpack.config.prod.js

```diff
@@ -31,6 +32,10 @@ const config = {
       PRODUCTION: JSON.stringify(true),
       VERSION: JSON.stringify(version),
     }),
+    new SubresourceIntegrityPlugin({
```

Comment:
Shouldnt we disable this in devmode?

Reply:
This is only triggered for prod? so in dev mode it wont call this as we have .dev.js config?